plugin.go

```diff
@@ -77,6 +79,10 @@ func (p Plugin) Exec() error {
 		}
 	}
 
+	if p.Config.EnvFile != "" {
+		_ = godotenv.Load(p.Config.EnvFile)
```

Comment:
I saw you noted in PR descrip that this needs to be loaded from plugin `exec` now ... why is that?  Would this not work if it remained where it was?  Do you have some sample issues you ran into otherwise?

Reply:
You are right, technically I didn't need to change it to solve the problem. But I'm thinking now, it make sense to me for 2 reasons:
- updating the Config struct on plugin.go to have the EnvFile param like other params
- On main.go we basically parse the input, and most of the logic we execute on plugin.go, so  might be good keep the env loading on plugin.go along with the assumeRole logic because they are kind of related.

What do you think?